lib/breathe/client.rb

```diff
@@ -1,43 +1,45 @@
 module Breathe
   class Client
+    attr_reader :api_key, :last_response
+
     BASE_URL = "https://api.breathehr.com/v1/"
 
     def initialize(api_key:)
       @api_key = api_key
     end
 
     def absences
-      @_absences ||= Absences.new(self)
-    end
-
-    def get(url, url_opts = {})
-      request(:get, url, url_opts)
+      @absences ||= Absences.new(self)
     end
 
-    private
+    def response(type, path, args)
```

Comment:
I do think named parameters are worth doing to protect against problems with parameter ordering and to make it a little clearer to the reader, removing the need to go and read the method to understand what each param means.

For example at the extreme of unhelpfulness we could have `foo("1", "", true)`. This is made much more manageable (IMO) when read as `foo(age: "1", name: "", teeth: true)`.. "Okay I need to provide a name! Duh"